examples/svelte-example/src/App.svelte

```diff
@@ -1,15 +1,17 @@
 <script lang="ts">
-	import { Dashboard, DashboardModal, DragDrop, ProgressBar } from "@uppy/svelte"
-	import Uppy from "@uppy/core"
+	import { Dashboard, DashboardModal, DragDrop, ProgressBar } from '@uppy/svelte'
+	import Uppy from '@uppy/core'
 	import Webcam from '@uppy/webcam'
 	import XHRUpload from '@uppy/xhr-upload'
 
-	let createUppy = () => Uppy().use(Webcam).use(XHRUpload, {
-		bundle: true,
-		endpoint: 'http://localhost:9967/upload',
-		metaFields: ['something'],
-		fieldName: 'files'
-	})
+	const createUppy = () => {
+		return new Uppy().use(Webcam).use(XHRUpload, {
```

Comment:
Out of curiosity, is there a reason to prefer `() => { return new Uppy }` over `() => new Uppy`?

Reply:
No reason, this was likely just eslint autofix